docs/source/docker/advanced/datasource_predictions.rst

```diff
@@ -0,0 +1,332 @@
+.. _ref-docker-datasource-predictions:
+
+Add Predictions to a datasource
+===============================
+
+Lightly can not only use images you provided in a datasource, but also predictions of a ML model on your images.
+They can be used e.g. for using Lightly for Active Learning or for using Lightly on object-level.
```

Comment:
used e.g. for using -> replace by "can be used for active learning for selecting images based on the objects in them."